pkg/homeutils/homeutils.go

```diff
@@ -0,0 +1,30 @@
+package homeutils
```

Comment:
This sounds too specific. I'd rather have `pathutils`. But that's not a blocker for this to land.